command/loadbot/loadbot_command.go

```diff
@@ -0,0 +1,213 @@
+package loadbot
+
+import (
+	"flag"
+	"fmt"
+	"github.com/0xPolygon/polygon-sdk/command/helper"
+	helperFlags "github.com/0xPolygon/polygon-sdk/helper/flags"
+	"github.com/0xPolygon/polygon-sdk/types"
+)
+
+type LoadbotCommand struct {
+	helper.Meta
+}
+
+func (l *LoadbotCommand) DefineFlags() {
+	if l.FlagMap == nil {
+		l.FlagMap = make(map[string]helper.FlagDescriptor)
+	}
+
+	l.FlagMap["tps"] = helper.FlagDescriptor{
+		Description: "Number of transactions executed per second by the loadbot",
+		Arguments: []string{
+			"TPS",
+		},
+		FlagOptional: true,
+	}
+
+	l.FlagMap["account"] = helper.FlagDescriptor{
+		Description: "Sets the account the use while running stress tests",
+		Arguments: []string{
+			"ACCOUNT",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+
+	l.FlagMap["gasLimit"] = helper.FlagDescriptor{
+		Description: "The specified gas limit",
+		Arguments: []string{
+			"LIMIT",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      true,
+	}
+
+	l.FlagMap["gasPrice"] = helper.FlagDescriptor{
+		Description: "The gas price",
+		Arguments: []string{
+			"GASPRICE",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      true,
+	}
+
+	l.FlagMap["url"] = helper.FlagDescriptor{
+		Description: "The URL used to make the transaction submission",
+		Arguments: []string{
+			"URL",
+		},
+		ArgumentsOptional: false,
+		FlagOptional:      false,
+	}
+
+	l.FlagMap["chain-id"] = helper.FlagDescriptor{
```

Comment:
Consider renaming this flag to chainid, since we have it without the dash in the rest of the codebase

Reply:
Fixed in https://github.com/0xPolygon/polygon-sdk/pull/197/commits/45b67ce0874d8dbd2120ee3c409098b5a1f99c15